src/components/activity-completion/completion-page-content.tsx

```diff
@@ -31,8 +32,27 @@ interface IProps {
 export const CompletionPageContent: React.FC<IProps> = (props) => {
   const { activity, activityName, onPageChange, showStudentReport,
     sequence, activityIndex, onActivityChange, onShowSequence } = props;
-
   const [answers, setAnswers] = useState<WrappedDBAnswer[]>();
+  const [feedback, setFeedback] = useState<ActivityFeedback | null>(null);
+
+  useEffect(() => {
+    watchAllAnswers(answerMetas => {
+      setAnswers(answerMetas);
+    });
+  }, []);
+
+  useEffect(() => {
+    if (activity.id) {
+      const unsubscribe = watchActivityLevelFeedback(fbs => {
```

Comment:
This same code is duplicated in [introduction-page-content.tsx](https://github.com/concord-consortium/activity-player/pull/505/files#diff-d3c73932daa88425106e6efa56d2aad5f2e925634fc0c81237171324739e1019) - it would be good to make this a shared utility function I think.

Reply:
I'm going to leave it for now because Ethan did create a shared utility function in a PR that's upcoming.